src/css/molecules/site-search-form.scss

```
@@ -0,0 +1,18 @@
+.site-search-form {
+  margin-bottom: $spacing-xl;
+
+  input.site-search-field {
+    background-image: url('/static/img/icons/search-black.svg');
+    background-position: 3px 3px;
+    background-repeat: no-repeat;
```

Is there a specific reason we write it out instead of using shorthand?

```suggestion
    background: transparent url('/static/img/icons/search-black.svg') 3px 3px no-repeat;
```